src/Tizen.NUI.Components/Controls/Popup.cs

```diff
@@ -331,7 +331,32 @@ public void AddContentText(View childView)
         /// Get style of popup.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public new PopupStyle Style => ViewStyle as PopupStyle;
+        //public new PopupStyle Style => ViewStyle as PopupStyle;
```

Comment:
Hello, 
please remove this if it is useless

Reply:
Removed